tests/integration/test_optislang/test_optislang_integration.py

```diff
@@ -38,66 +38,60 @@ def test_simple_solve(load_mixing_elbow_param_case_dat):
-
-        # check if solution is steady or transient
-        workflow = solver_session.scheme_eval.string_eval("(rp-unsteady?)")
-
-        # iterate workflow
-        if workflow == "#t":
-            solver_session.tui.solve.dual_time_iterate()
-        else:
-            solver_session.tui.solve.iterate()
-
-        convergence = solver_session.scheme_eval.string_eval(
-            "(rpgetvar 'solution/converged?)"
-        )
+    # Step 4: Set a variation on these input parameters
+    # variations/designs are generated by optiSLang based on
+    # algorithm selected
+    from ansys.fluent.parametric.parameters import InputParameters 
+    inputs_table = InputParameters(solver_session)
+    if 'inlet2_temp' in input_parameters.keys(): 
+        inputs_table['inlet2_temp'] = 500 
+    solver_session.tui.file.write_case("design_elbow_param.cas.h5")
+
+    # Step 5: Solve
+    solver_session.tui.solve.initialize.initialize_flow()
+
+    # check if solution is steady or transient
+    workflow = solver_session.scheme_eval.string_eval("(rp-unsteady?)")
+
+    # iterate workflow
+    if workflow == "#t":
+        solver_session.tui.solve.dual_time_iterate()
+    else:
+        solver_session.tui.solve.iterate()
+
+    convergence = solver_session.scheme_eval.string_eval(
+        "(rpgetvar 'solution/converged?)"
+    )
 
-        # solution output (test conditional statement)
-        if convergence == "#f":  # -> let user know
-            print("Failed to converge")
-        else:
-            print("Solution is converged")
+    # solution output (test conditional statement)
+    if convergence == "#f":  # -> let user know
+        print("Failed to converge")
+    else:
+        print("Solution is converged")
 
-        assert convergence == "#t", "Solution failed to converge"
+    assert convergence == "#t", "Solution failed to converge"
 
-        # Step 6: Read the data again from the case and data file
-        solver_session.tui.file.read_case_data(case_path)
-        input_parameters2 = solver_session.scheme_eval.eval(
-            (Symbol("list-input-parameters"),)
-        )
-        output_parameters2 = solver_session.scheme_eval.eval(
-            (Symbol("list-output-parameters"),)
-        )
-        input_parameters2 = input_parameters2["inlet2_temp"]
-        output_parameters2 = output_parameters2["outlet_temp-op"]
-        assert input_parameters[0] == input_parameters2[0]
-        assert output_parameters[0] == output_parameters2[0]
-        solver_session.exit()
+    # Step 6: Read the data again from the case and data file
+    solver_session.tui.file.read_case_data(case_path)
+    from ansys.fluent.parametric.parameters import InputParameters, OutputParameters
+    inputs_table = InputParameters(solver_session)
+    outputs_table = OutputParameters(solver_session) 
+
+    output_units = outputs_table["outlet_temp-op"].split(' ')[1].strip('[]')
+    assert input_parameters["inlet2_temp"][0] == inputs_table["inlet2_temp"]
```

Comment:
input_parameters consists of the original value of parameters and inputs_table contains the value that was set before solving. why would we expect these two values to be the same? Rather following check can be performed:
assert inputs_table["inlet2_temp"] = 500 ---> this is to confirm that the value with which we solved the case file is as expected